scripts/setup_olserver.sh

```diff
@@ -18,9 +18,35 @@ fi
 
 # apt list --installed
 sudo apt-get update
-sudo apt-get install -y docker.io docker-compose
-docker --version        # 19.03.8
-docker-compose version  #  1.25.0
+# Remove any old versions and install newer versions of Docker Engine and Docker Compose.
+# See https://docs.docker.com/engine/install/ubuntu/ for any possible changes.
+sudo apt-get remove docker docker-engine docker.io containerd runc
+DOCKER_VERSION=5:20.10.7~3-0~ubuntu-focal
```

Comment:
This needs to be updated but I am unclear on how to create `DOCKER_VERSION` that matches our [current Docker versions](https://github.com/internetarchive/openlibrary/issues/7676#issuecomment-1480274108).

See #7706

Reply:
In ```scripts/setup_olserver.sh``` it is ```5:20.10.7~3-0~ubuntu-focal``` but in ```scripts/solr_builder/Dockerfile``` it is ```5:20.10.23~3-0~ubuntu-jammy```. Is it supposed to be like that ? And here ([Current Docker Verisons](https://github.com/internetarchive/openlibrary/issues/7676#issuecomment-1480274108)) it is ```23.0.1```.

So we need to change it to ```23.0.1``` ?

Reply:
Yep switch to `5:23.0.6-1~ubuntu.20.04~focal` According to chatgpt, `apt-cache madison docker-ce` displays all possible versions.